buildscripts/templates/SemanticAttributes.java.j2

```diff
@@ -97,6 +97,30 @@ public final class {{class}} {
 
     {%- endfor %}
 
+    {%- if class_name == "MessagingOperationValues" %}
```

Comment:
I don't see any evidence in the template that we've accommodated the removal of values from semantic conventions enums, which manifest as static inner classes. But this seemed like the most correct thing to do. Also, we're using japicmp to ensure we don't make incompatible changes (until we resolve #6) and not fixing this causes japicmp to fail.